src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -37,6 +37,7 @@ class CompilerContainer {
     }
     if (!this._view.compilationButton) return
     const button = this.compilationButton(name.split('/').pop())
+    this._disableCompileBtn(!name || name === '')
```

Comment:
if `name === '' // true` then `!name // true`. So you don't need `|| name === ''`.
In Javascript : 
`!0 // true`
`!"" // true`
`!null // true`
`!undefined // true`
BUT
`![] // false`
`!{} // false`